bip-0009.mediawiki

```diff
@@ -55,10 +61,8 @@ visible.
     }
 
 '''Success: Activation Delay'''
-The consensus rules related to ''locked-in'' soft fork will be enforced in
-the second retarget period; ie. there is a one retarget period in
-which the remaining 5% can upgrade.  At the that activation block and
-after, miners should stop setting bit B, which may be reused for a different soft fork.
+The consensus rules related to ''locked-in'' soft fork will be enforced in the second retarget period; ie. there is a one retarget period in
+which the remaining 5% can upgrade.  Upon reaching the end of the period, the soft fork is considered active and miners SHALL enforce the new rules beginning with the next block whose height is divisible by 2016. At this point, miners SHALL stop setting bit B which MAY now be reused for a different soft fork.
```

Comment:
Eric Lombrozo <notifications@github.com> writes:
>> @@ -55,10 +61,8 @@ visible.
>>      }
>>  
>>  '''Success: Activation Delay'''
>> -The consensus rules related to ''locked-in'' soft fork will be enforced in
>> -the second retarget period; ie. there is a one retarget period in
>> -which the remaining 5% can upgrade.  At the that activation block and
>> -after, miners should stop setting bit B, which may be reused for a different soft fork.
>> +The consensus rules related to ''locked-in'' soft fork will be enforced in the second retarget period; ie. there is a one retarget period in
>> +which the remaining 5% can upgrade.  Upon reaching the end of the period, the soft fork is considered active and miners SHALL enforce the new rules beginning with the next block whose height is divisible by 2016. At this point, miners SHALL stop setting bit B which MAY now be reused for a different soft fork.
>
> In the RFC they are synonymous, I have no strong preference either way. MUST is fine.

Indeed; if you can stick with the same subset throughout it makes life
easier for the reader.

Thanks,
Rusty.